app/models/post.rb

```diff
@@ -2,4 +2,15 @@ class Post < ApplicationRecord
   belongs_to :author, class_name: 'User'
   has_many :comments
   has_many :likes
+  def update_posts_counter
+    author.update(posts_counter: author.posts.count)
+  end
+
+  def recent_comments(limit = 5)
+    comments.order(created_at: :desc).limit(limit)
+  end
+
+  def update_likes_counter
+    update(likes_counter: likes.count)
+  end
```

Comment:
- [ ] [OPTIONAL] can use one of the [Rails Active Callbacks](https://guides.rubyonrails.org/active_record_callbacks.html) - after_save to call the `#update_post_counter` method every time a new post gets saved to database, then make the `update_post_counter` method private.
**NB** Please do same for the `update_likes_counter` method 👍